common/client.go

```diff
@@ -35,6 +37,10 @@ func (c *Client) Send(request Request, response Response) (err error) {
 		httpRequest.Header["Content-Type"] = []string{"application/x-www-form-urlencoded"}
 	}
 	log.Printf("[DEBUG] http request=%v", httpRequest)
+
+	command, _ := http2curl.GetCurlCommand(httpRequest)
```

Comment:
here we introduced a new dependency, so I think we should add something like vendor.json to record it

Reply:
i dont think this repo use govendor. 
I think it's better to setup govendor in separate PR?

Reply:
it because it has no dependency yet...

Reply:
done